src/core/generator/clones/clonesGenerator.ts

```diff
@@ -107,7 +114,12 @@ export const generateConfiguredClones = async (
   }
 };
 
-/** Checks if there are any custom clones to be created */
+/**
+ * Checks if there are any custom clones to be created.
+ *
+ * @param {Config} config - The new configuration that customizes the icons and the manifest.
+ * @returns {boolean} True if there are custom clones to be created, false otherwise.
```

Comment:
This might be opinionated but from my point of view the function name "hasCustomClones" and the return type "boolean" are very self-explanatory, that "custom clones are to be created" result to "true" and otherwise "false". A good comment should add some additional explanation to the existing code, and not just repeating the same information which is already existing in the code. Otherwise it will blow up the code base, without bringing any value and it will increase the effort to maintain the code.